internal/acceptance/default_namespace_test.go

```diff
@@ -0,0 +1,34 @@
+package acceptance
```

Comment:
Could we add a test case that deletes an existing setting? The verification is to call GET and get back NOT_FOUND after deletion

Reply:
Added